tests/cmdline/commands/test_code.py

```diff
@@ -271,6 +271,16 @@ def test_code_export(run_cli_command, aiida_code_installed, tmp_path, file_regre
     )
     filepath = tmp_path / 'code.yml'
 
+    # Run export without explicit output_file path
+    # Don't provide output_file argument, but still export in tmp_path to avoid pollution of main repo when running test
+    cwd = os.getcwd()
+    os.chdir(tmp_path)
+    options = [str(code.pk)]
+    run_cli_command(cmd_code.export, options)
+    assert filepath.is_file
+    filepath.unlink()
+    os.chdir(cwd)
```

Comment:
I think this is best accomplished by using the `chdir_tmp_path` fixture instead

Reply:
Done!